qa/tasks/cephfs/caps_helper.py

```diff
@@ -113,6 +114,8 @@ def _gen_mds_cap_str(c):
     return mds_cap
 
 
+# XXX: Inheriting CephFSTestCase ONLY to get methods like self.assertEqual(),
```

Comment:
These helpers are methods (not functions), so possible to inherit them directly.

Reply:
If they can be imported, it makes sense to do that?

Reply:
What helpers are being used from CephFSTestCase? FWIW, assertEqual can be imported from unittest.TestCase.

Reply:
If I am not mistaken in understanding your comment, assertEqual() is method (and not a function) in class unittest.TestCase and therefore it can't be imported without importing and initializing an instance of class unittest.TestCase.

Reply:
Understood. I find it a bit heavy-weight to import CephFSTestCase just to get assertEqual.

Reply:
It import around 42 such methods and we do so in almost every class.

Reply:
Discussion from IRC

```
<overclk> If its just one helper that's required, can't we just have a custom assert equals?
<rishabh> that would amount to code duplication. and we import CephFSTestCase in every class in qa/tasks/cephfs, so i don't think it's a problem.
<overclk> rs
<overclk> rishabh: 
<overclk> qa/tasks/cephfs/test_data_scan.py:    def assert_equal(self, a, b):
<overclk> this test already has a custome defn, maybe move this to a more usable place and reuse it in your pr?
<rishabh> okay but isn't it bad idea to duplicate code?
<overclk> it is
<overclk> but again, caps_helper is supposed to be providing helper routines
<overclk> subclassing from CephFSTestCase just to get one helper seems a bit off-putting...
<rishabh> i agree.
<rishabh> i wish there was an way to import those methods
<rishabh> but there isn't any way to do so.
<overclk> see what test_data_scan does
<overclk> and if there that can be improved
<overclk> not sure why it has a custom defn
```

Reply:
@rishabh-d-dave lets keep it simple and just use plain vanilla assert :)

Reply:
I deliberately avoided this option before because assert*() methods from unittest.TestCase are much more elegant. These methods print a well-formatted message when assert fails. So current way is better IMO.

Reply:
Class CephFSTestCase have been imported by CapTester much before this PR.  Replacing now is a little outside the scope of this PR.

Reply:
I've updated the code. PTAL.

Simply using assert keyword wasn't possible. I need to write a couple of extra lines to set a message that can be passed to assert keyword. The following codeblock repeated few times in caps_helper -

``
msg = '''abcd'''
assert x == y, msg
```

So I instead created a method `assert_equal()` instead.

And in case of `assert x in y`, the message was really long. Too long to be non-distracting inside a for loop. So for the sake of simplicity, readability and sucicntness, I create method assert_in().